src/assets/js/data/Api.js

```diff
@@ -197,6 +197,13 @@ export const deleteAdmin = (adminId) =>
     .set('Authorization', cookies.get(CookieTypes.admin.token, {path: '/'}))
     .use(apiPrefix));
 
+
+export const deleteUser = (eventAlias, id) =>
+  promisify(request
```

Comment:
Elsewhere on the website we only use GET and POST requests. I think a DELETE is fine, but it's the first one to be different.